packages/backend-core/src/middleware/passport/oidc.js

```diff
@@ -89,11 +91,24 @@ function validEmail(value) {
  * from couchDB rather than environment variables, using this factory is necessary for dynamically configuring passport.
  * @returns Dynamically configured Passport OIDC Strategy
  */
-exports.strategyFactory = async function (config, callbackUrl, saveUserFn) {
+exports.strategyFactory = async function (config, saveUserFn) {
```

Comment:
It's slightly confusing that the `config` here isn't the config loaded from the db, rather it's an enriched form of the config that's aware of the oidc elements from the provider. Maybe a different name could help here like `strategyConfig` which makes it more clear that this argument is the result of the below function

Reply:
Update incoming